### ISSUE TITLE
JitsiMeetEventListener() is not working when end room by clear all screen.

### ISSUE THREAD

**Issue by bunthoeun-develop** [jitsi/jitsi-meet-flutter-sdk#62]
Actual behavior: When I end call by clear call screen, JitsiMeetEventListener() is not working.
Expected behavior: When I end call by clear call screen, JitsiMeetEventListener() should working.

https://github.com/jitsi/jitsi-meet-flutter-sdk/assets/65479486/22f698d9-7ba4-474f-845f-b49ae345cec4

<img width="407" alt="Screenshot 2024-05-09 at 4 26 16 in the afternoon" src="https://github.com/jitsi/jitsi-meet-flutter-sdk/assets/65479486/1fe29986-bf18-4391-b3f1-0e54410243cc">

**Comment by saghul** [comment]
Not sure I understand. You are swiping out the JitsiMeetActivity, right? There is very little time to do anything wheen the activity is destrooyed. What do you exppect it to happen?

**Comment by bunthoeun-develop** [comment]
Want to detect when I leave conference room. if end by bottom(icon end on UI), then I can use readyToClose() event.

**Comment by saghul** [comment]
> Want to detect when I leave conference room. if end by bottom(icon end on UI), then I can use readyToClose() event.

Yes, that is correct. But if you swipe the activity out there is no way for the meeting to tell you. You'll need to listen to activity lifecycle events.
